henryPuma/assignments/treesDFS/lc112/lc112.js

```diff
@@ -0,0 +1,90 @@
+// Problem Statement #
+
```

Comment:
Nice work here! Could also be helpful to write some additional test cases, especially for any edge cases you identify.